app/src/main/java/woowacourse/movie/activity/MovieReservationActivity.kt

```diff
@@ -0,0 +1,105 @@
+package woowacourse.movie.activity
+
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import woowacourse.movie.R
+import woowacourse.movie.domain.Movie
+import woowacourse.movie.getSerializable
+import woowacourse.movie.view.Counter
+import woowacourse.movie.view.DateSpinner
+import woowacourse.movie.view.MovieController
+import woowacourse.movie.view.ReservationButton
+import woowacourse.movie.view.SaveStateCounter
+import woowacourse.movie.view.SaveStateSpinner
+import woowacourse.movie.view.TimeSpinner
+
+class MovieReservationActivity : AppCompatActivity() {
+    private val counter: SaveStateCounter by lazy {
+        SaveStateCounter(
+            Counter(
+                findViewById(R.id.movie_reservation_people_count_minus),
+                findViewById(R.id.movie_reservation_people_count_plus),
+                findViewById(R.id.movie_reservation_people_count),
+                INITIAL_COUNT,
+            ),
+            COUNTER_SAVE_STATE_KEY,
```

Comment:
```suggestion
            counter = Counter(
                minus = findViewById(R.id.movie_reservation_people_count_minus),
                plus = findViewById(R.id.movie_reservation_people_count_plus),
                text = findViewById(R.id.movie_reservation_people_count),
                count = INITIAL_COUNT,
            ),
            saveStateKey = COUNTER_SAVE_STATE_KEY,
```
코틀린에서는 argument name이라는 것을 지원합니다.
argument name을 활용하면, 전달하는 값이 어떤 값인지 쉽게 알 수 있어요. (이하 관련 내용 동일)

Reply:
default parameter가 적용된 많은 수의 인자를 전달해주는 부분에서 argument name을 활용해주었습니다.